src/controllers/tasks/active.officers.details.controller.ts

```diff
@@ -52,7 +53,7 @@ const buildCorporateOfficerList = (officers: ActiveOfficerDetails[], wantedOffic
       return {
         dateOfAppointment: officer.dateOfAppointment,
         forename: formatTitleCase(officer.foreName1),
-        identificationType: officer.identificationType,
+        identificationType: lookupIdentificationType(officer.identificationType as string),
```

Comment:
might be better to use the ternary operator here '?'. Not sure of the behaviour if identificationType is not present (it is optional) and we convert it to a string.
perhaps use identificationType ? lookup... : ""

Reply:
Might not be easy to test for this given the way we test the page at atm. We are just checking the while page for specific field but checking for the expected "" could just lead to false positives.

Reply:
Agreed, if we are testing based on the page content, but could you possibly mock out the 'lookupIdentificationType' function using Jest in a specific test in which the API returns a null and then check that the mock is NOT called?